coda/config/wsgi.py

```diff
@@ -14,15 +14,15 @@
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
 ENV = os.path.join(BASE_DIR, 'env')
-SITE_PACKAGES = os.path.join(ENV, '/lib/python2.7/site-packages')
+SITE_PACKAGES = os.path.join(ENV, '/lib/python3.7/site-packages')
 
 site.addsitedir(SITE_PACKAGES)
 sys.path.append(os.path.join(BASE_DIR, 'coda/'))
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings.production')
 
 activate_env = os.path.join(ENV, 'bin/activate_this.py')
-execfile(activate_env, dict(__file__=activate_env))
+exec(activate_env, dict(__file__=activate_env))
```

Comment:
I am not sure that `exec` and `execfile` are an exact replacement. 
I think in other Django projects we are doing:
```
with open(activate_env) as activate_f:
    exec(compile(activate_f.read(), activate_env, 'exec'), dict(__file__=activate_env))
```

Reply:
Well...I wasn't sure about this change myself. But since the setting worked fine and the app was working I thought I'll let it be there. But I can change it to what you have suggested if we are following that in other django projects.

Reply:
I don't think this file gets used in dev/Docker environment, in which case you wouldn't see it not working. If I open a Python 3 interpreter and run `exec(activate_env, dict(__file__=activate_env))` where `activate_env` is a string path as it is above, I get an exception. [The docs](https://docs.python.org/3.3/whatsnew/3.0.html?highlight=execfile#builtins) indicate the first argument should be a file object in this case.